third_party/docfx-doclet-143274/src/main/java/com/microsoft/build/YmlFilesBuilder.java

```diff
@@ -115,8 +145,10 @@ void process() {
           }
         }
       }
-      // build project summary page
-      projectBuilder.buildProjectMetadataFile(packageItems, projectMetadataFile);
+      // build project summary page if disableLibraryOverview=true

```

Comment:
Likely same question as before -- why is the project summary page dependent on the library overview not being generated?

Reply:
Same as above - This was the logic to overwrite the current existing overview with the new overview file, assuming the `disable-libraryOverview` flag is false). Let me know though if you have a different approach in mind for doing so.